README.md

```diff
@@ -394,6 +394,42 @@ type Output struct {
 }
 ```
 
+### Error Wrapping
+
+With `goverter:wrapErrors`, goverter will wrap conversion errors with additional troubleshooting information.
+The tag `goverter:wrapErrors` does not take parameters. The `wrapErrors` flag can also be set via command line.
+
+```go
+// goverter:converter
+// goverter:extend strconv:Atoi
+type Converter interface {
+    // goverter:wrapErrors
```

Comment:
You probably use the CLI flag to enable this behavior, right? I think wrapErrors should be moved to a converter comment to enable it for the whole converter.

Reply:
I have the flag controlled globally as well as per Method (not converter). I can move the method one to the converter level, is this what you ask for?

Reply:
Yes.

Reply:
ok, I added wrapErrors on converter level too, now it can be controlled on any level